tests/test_tuya_sensor.py

```diff
@@ -115,7 +116,10 @@ async def test_handle_get_data_enum_batt(
     assert ep.tuya_manufacturer is not None
     assert isinstance(ep.tuya_manufacturer, TuyaMCUCluster)
 
-    message = b"\x09\xe0\x02\x0b\x33\x01\x02\x00\x04\x00\x00\x00\xfd\x02\x02\x00\x04\x00\x00\x00\x47\x04\x02\x00\x04\x00\x00\x00\x01"
+    if model == "_TZE204_upagmta9":  # Uses dp 3 for battery
+        message = b"\x09\xe0\x02\x0b\x33\x01\x02\x00\x04\x00\x00\x00\xfd\x02\x02\x00\x04\x00\x00\x00\x47\x03\x02\x00\x04\x00\x00\x00\x01"
+    else:
+        message = b"\x09\xe0\x02\x0b\x33\x01\x02\x00\x04\x00\x00\x00\xfd\x02\x02\x00\x04\x00\x00\x00\x47\x04\x02\x00\x04\x00\x00\x00\x01"
```

Comment:
Hmm. I feel like this could be a bit cleaner.
Not sure if this works, but can we put both of these in a constant, then add another parameter to `parametrize` like `message`, then set a default for that in the method definition (to the "normal" message, and only have `_TZE204_upagmta9` provide the dp 3 constant?